packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -326,6 +333,61 @@ function parseBaseElement(
     return element;
 }
 
+function parseLwcElement(
+    ctx: ParserCtx,
+    parse5Elm: parse5.Element,
+    parsedAttr: ParsedAttribute,
+    parse5ElmLocation: parse5.ElementLocation
+) {
+    let lwcElementParser;
+
+    switch (parse5Elm.tagName) {
```

Comment:
Is there a reason to use a `switch` here rather than `if (parse5Elm.tagName === LwcTagName.Component) { ... } else { ... }`? Do you expect there will be additional cases here?

Reply:
Yup, we're planning to add more LWC managed elements (that start with `lwc:*`) like [`lwc:host`](https://github.com/salesforce/lwc-rfcs/pull/72)